community/kernel/src/main/java/org/neo4j/kernel/NeoStoreDataSource.java

```diff
@@ -1123,6 +1142,14 @@ public synchronized void stop()
         // fail since the logFile no longer works.
     }
 
+    private void awaitAllTransactionsClosed()
+    {
+        while ( !neoStoreModule.neoStore().closedTransactionIdIsOnParWithOpenedTransactionId() )
+        {
+            LockSupport.parkNanos( 1_000_000 ); // 1 ms
```

Comment:
I have simply moved that method from 'LogRotationControl'. We have that today in the code: https://github.com/neo4j/neo4j/blob/2.3/community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/log/rotation/LogRotationControl.java#L50

I am not against fixing it, but doing it in this PR feels wrong.